### DISCUSSION TITLE
Vertical scrollbar and "Show Pages Continuously" (or not)

### DISCUSSION THREAD

**Discussion by dumb0x** [sumatrapdfreader/sumatrapdf#2430] [Ideas]
As a new Sumatra user, I was so struck by its excellence (for example, the only PDF reader I have found that gets bookmarks right) that I was quick to contribute -- figuring that anything this good should be able to work out little problems easily. Then I found an issue raised by a number of users in a 2015 thread ["Display scrollbar in Single Page view" ](https://forum.sumatrapdfreader.org/t/display-scrollbar-in-single-page-view/1135)where there seems a strange mental block about a simple concept, seeming to elude easy resolution. That problem remains but the thread is old and closed, so I resurrect it here.

The concept is simple: What does the vertical scrollbar _represent_? Is it the progress on a page or the progress through the document? _It is obviously the latter_, in all readers or editors (of PDF or any other document type) that I know, including Sumatra. Yet Sumatra alone seems to make the mistake of saying that the scrollbar is not relevant unless "Show Pages Continuously" (what other readers call "Enable Scrolling") is enabled.  It's just weird, and a disservice to a great reader.

Now, it's true that there is always the non-graphical page count for reference regarding position in the document, but that's also true when "Show Pages Continuously" is enabled -- in which case the scrollbar shows position in the document, not position on the page. Position in the document remains a concern regardless how pages are displayed, so why would that graphical representation be removed when  "Show Pages Continuously" is disabled?

**Comment by GitHubRulesOK** [comment]
Forum links are generally needing changes due to discourse redirection
it better to link to the local dead issue https://github.com/sumatrapdfreader/sumatrapdf/issues/191

**Reply by dumb0x** [reply]
Thanks. I was (obviously) not aware of that redirection (and still don't understand the reason for it (unless it's an excess of caution)). So I appreciate you providing a better link -- but it's _not a link to the **same** thread_ ("Display scrollbar in Single Page view"); rather to another thread ("Scrollbar disappears in "Fit a Single Page" mode" (BTW: _referenced_ in that thread)) with the same points being made, with some imprecision (about "Single Page") and much misunderstanding (about the purpose of a vertical scrollbar).  

It looks like the thread I quoted ("Display scrollbar in Single Page view" (https://forum.sumatrapdfreader.org/t/display-scrollbar-in-single-page-view/1135)) did not even make it into the GitHub project issues.  That is too bad (mostly because nothing in forum.sumatrapdfreader.org seems to be dated), but IAC it should be instructive that nearly identical threads cover the same issue. Thank you for connecting two such threads; a look through the issues indicates there are many on the same topic. Doesn't that suggest that much time could be saved by simply grasping this simple concept: What is the purpose of a vertical scrollbar?  

When we understand that a vertical scrollbar is a graphical reference to position _in the document_ (not position on a page), then the confusion about not needing it when "Show Pages Continuously" is disabled goes away. It is needed, or not needed, regardless of the state of "Show Pages Continuously".
